src/s3path.jl

```diff
@@ -193,6 +193,37 @@ function Base.isdir(fp::S3Path)
     return iterate(objects) !== nothing
 end
 
+function FilePathsBase.walkpath(fp::S3Path; topdown=true, follow_symlinks=false, onerror=throw)
+
+    # Select objects with that prefix
```

Comment:
remind me the `prefix` here refers to which parameter?

Reply:
It's the combination of the `fp.bucket` and `fp.key`, which is the path to the root directory you want to start searching in, e.g. `s3://mybucket/path/to/root`